install/deploy_final_non-compute_node.md

```diff
@@ -87,25 +87,29 @@ The steps in this section load hand-off data before a later procedure reboots th
 
 1. (`pit#`) Upload NCN boot artifacts into S3.
 
-    ```bash
-    set -o pipefail
-    kubernetes_rootfs="$(readlink -f /var/www/ncn-m002/rootfs)" &&
-    kubernetes_initrd="$(readlink -f /var/www/ncn-m002/initrd.img.xz)"  &&
-    kubernetes_kernel="$(readlink -f /var/www/ncn-m002/kernel)"  &&
-    kubernetes_version="$(basename ${kubernetes_rootfs} .squashfs | awk -F '-' '{print $NF}')" &&
-    ceph_rootfs="$(readlink -f /var/www/ncn-s001/rootfs)" &&
-    ceph_initrd="$(readlink -f /var/www/ncn-s001/initrd.img.xz)" &&
-    ceph_kernel="$(readlink -f /var/www/ncn-s001/kernel)" &&
-    ceph_version="$(basename ${ceph_rootfs} .squashfs | awk -F '-' '{print $NF}')" &&
-    cray artifacts create boot-images "k8s/${kubernetes_version}/rootfs" "${kubernetes_rootfs}" &&
-    cray artifacts create boot-images "k8s/${kubernetes_version}/initrd" "${kubernetes_initrd}" &&
-    cray artifacts create boot-images "k8s/${kubernetes_version}/kernel" "${kubernetes_kernel}" &&
-    cray artifacts create boot-images "ceph/${ceph_version}/rootfs" "${ceph_rootfs}" &&
-    cray artifacts create boot-images "ceph/${ceph_version}/initrd" "${ceph_initrd}" &&
-    cray artifacts create boot-images "ceph/${ceph_version}/kernel" "${ceph_kernel}" && echo SUCCESS
-    ```
+    1. Upload Kubernetes NCN artifacts
 
-    Ensure that the output from the above command chain ends with `SUCCESS`.
+        ```bash
+        set -o pipefail
+        IMS_UPLOAD_SCRIPT=$(rpm -ql docs-csm | grep ncn-ims-image-upload.sh) &&
+        export IMS_ROOTFS_FILENAME="$(readlink -f /var/www/ncn-m002/rootfs)" &&
+        export IMS_INITRD_FILENAME="$(readlink -f /var/www/ncn-m002/initrd.img.xz)"  &&
+        export IMS_KERNEL_FILENAME="$(readlink -f /var/www/ncn-m002/kernel)"  &&
+        K8S_IMS_IMAGE_ID=$($IMS_UPLOAD_SCRIPT) &&
+        echo SUCCESS
```

Comment:
Of course we expect the script to fail if it doesn't provide the image ID in the output, but maybe add this code to validate that the output is not empty, and also display the output to the user.

```suggestion
            export IMS_ROOTFS_FILENAME="$(readlink -f /var/www/ncn-m002/rootfs)" &&
            export IMS_INITRD_FILENAME="$(readlink -f /var/www/ncn-m002/initrd.img.xz)"  &&
            export IMS_KERNEL_FILENAME="$(readlink -f /var/www/ncn-m002/kernel)"  &&
            K8S_IMS_IMAGE_ID=$($IMS_UPLOAD_SCRIPT) &&
            [[ -n ${K8S_IMS_IMAGE_ID} ]] &&
            echo -e "Kubernetes NCN image IMS ID: ${K8S_IMS_IMAGE_ID}\nSUCCESS"
```